.github/workflows/update.yml

```
@@ -18,12 +18,18 @@ jobs:
         with:
           name: niri
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-      - name: Update flake.lock
+      - name: Update versions
+        run: |
+          nix run nixpkgs#nix-update -- --flake --version=stable --override-file=pkgs/niri/stable/default.nix niri-stable
+          nix run nixpkgs#nix-update -- --flake --version=branch --override-file=pkgs/niri/unstable/default.nix niri-unstable
+          nix run nixpkgs#nix-update -- --flake --version=stable --override-file=pkgs/xwayland-satellite/stable/default.nix xwayland-satellite-stable
+          nix run nixpkgs#nix-update -- --flake --version=branch --override-file=pkgs/xwayland-satellite/unstable/default.nix xwayland-satellite-unstable
```

nix-update should *not* be automatically updating the stable packages here

or at least, not for niri. maybe for xwayland-satellite? haven't thought too much about this.

but, generally, i ensure that the settings module always exhaustively supports `niri-stable`. it must therefore only be manually updated.

it would be nice if the xwayland-satellite package was automatically updated but not automatically merged? so i get a nixpkgs-like workflow where the bot pings me "hey, this can be updated pretty please". niri could behave like this too. but this should not be in the automatically-merged workflow.

ultimately, that's for me to figure out how exactly i want to deal with it. since i'm the one who'll be manually merging the stable updates. so for now, just remove the lines updating the stable packages.

```suggestion
          nix run nixpkgs#nix-update -- --flake --version=branch --override-file=pkgs/niri/unstable/default.nix niri-unstable
          nix run nixpkgs#nix-update -- --flake --version=branch --override-file=pkgs/xwayland-satellite/unstable/default.nix xwayland-satellite-unstable
```